### ISSUE TITLE
Umlauts

### ISSUE THREAD

**Issue by drjackyl** [khalim19/gimp-plugin-export-layers#26]
I get an error when trying to start "Export Layers..." with a file containing Umlauts and sharp S in its path.

Mac OS X 10.10.5
GIMP 2.8.14p2 (build from lisanet.de)
Modified GIMP.app/Contents/Resources/lib/gimp/2.0/interpreters/pygimp.interp for Python 2.7

Traceback:

```
Traceback (most recent call last):
  File "/Applications/GIMP.app/Contents/Resources/lib/gimp/2.0/python/gimpplugin.py", line 58, in _run
    return apply(getattr(self, name), params)
  File "/Users/fl/Library/Application Support/GIMP/2.8/plug-ins/export_layers.py", line 111, in plug_in_export_layers
    self._run_export_layers_interactive(image)
  File "/Users/fl/Library/Application Support/GIMP/2.8/plug-ins/export_layers/pygimplib/pggui.py", line 339, in func_wrapper
    func(self, *args, **kwargs)
  File "/Users/fl/Library/Application Support/GIMP/2.8/plug-ins/export_layers.py", line 147, in _run_export_layers_interactive
    gui_plugin.export_layers_gui(image, self.settings, self.session_source, self.persistent_source)
  File "/Users/fl/Library/Application Support/GIMP/2.8/plug-ins/export_layers/gui_plugin.py", line 606, in export_layers_gui
    _ExportLayersGui(image, settings, session_source, persistent_source)
  File "/Users/fl/Library/Application Support/GIMP/2.8/plug-ins/export_layers/gui_plugin.py", line 228, in __init__
    _setup_image_ids_and_directories_and_output_directory(self.settings, self.image)
  File "/Users/fl/Library/Application Support/GIMP/2.8/plug-ins/export_layers/gui_plugin.py", line 184, in _setup_image_ids_and_directories_and_output_directory
    current_image, settings['gui_session']['image_ids_and_directories'].value[current_image.ID])
  File "/Users/fl/Library/Application Support/GIMP/2.8/plug-ins/export_layers/pygimplib/pgsetting.py", line 1059, in update_current_directory
    self.set_value(directory_for_current_image)
  File "/Users/fl/Library/Application Support/GIMP/2.8/plug-ins/export_layers/pygimplib/pgsetting.py", line 309, in set_value
    self._assign_and_validate_value(value)
  File "/Users/fl/Library/Application Support/GIMP/2.8/plug-ins/export_layers/pygimplib/pgsetting.py", line 442, in _assign_and_validate_value
    self._validate(value)
  File "/Users/fl/Library/Application Support/GIMP/2.8/plug-ins/export_layers/pygimplib/pgsetting.py", line 996, in _validate
    self._value_to_str(value) + '\n'.join([message for message in new_status_messages])
SettingValueError: "/Users/fl/Jump/Original Sondärzöüchnß": File path contains invalid characters.
```

Don't get me wrong, but I think in 2015 this should not be a problem :-) Don't know if it's a problem of the plugin, though.

Thanks for the plugin, saves me a lot of time!

Regards

**Comment by khalim19** [comment]
Thanks for the report. It should be fixed in the latest source revision - see if it works for you.

By the way, you're absolutely right, it should not be a problem these days. :P

**Comment by khalim19** [comment]
Fixed in 2.4.
